examples/pachyderm/sklearn_pachyderm.ipynb

```diff
@@ -0,0 +1,932 @@
+{
```

Comment:
<p>Would be good to show what's inside of get-data.py, maybe as a <code>head / tail -10</code>  if too large</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
It's a short file, just added viewing of the content.